packages/gatsby/src/schema/__tests__/kitchen-sink.js

```diff
@@ -54,8 +54,12 @@ describe(`Kitchen sink schema test`, () => {
     store.dispatch({
       type: `CREATE_TYPES`,
       payload: `
-        type PostsJson implements Node @infer {
-          id: String!
+        interface Post @nodeInterface {
```

Comment:
`@infer` does not have any effect on interfaces, because type inference works on node types which are in the store (which would be the types that implement the interface). so i kept `@infer` on `PostsJson`, but no need on `Post`.